tests/testthat/test_helpers.R

```diff
@@ -35,3 +35,32 @@ test_that("list_submissions returns correct output", {
   out <- list_submissions(adc, '2018-10-01', '2018-10-03')
   expect_equal(out$submitter_name[1], 'Rachel Obbard')
 })
+
+test_that("write a valid EML to the given path", {
+  cn <- dataone::CNode('PROD')
+  adc <- dataone::getMNode(cn, 'urn:node:ARCTIC')
```

Comment:
we should change this, this test would write a dummy package to production if a token is set. It should point to the test node

Reply:
@rachelsun97 I think that this issue is still not resolved. The cn and mn should be to test at the lines where this comment is written (40-41 in test_helpers.R) as well as below (which you changed below)